doc/book/helpers/body-parse.md

```diff
@@ -43,13 +43,17 @@ return [
         ],
     ],
     'middleware_pipeline' => [
-        'pre_routing' => [
-            [ 'middleware' => Helper\BodyParams\BodyParamsMiddleware::class ],
-            /* ... */
-        ],
-        'post_routing' => [
-            /* ... */
+        [ 'middleware' => Helper\BodyParams\BodyParamsMiddleware::class, 'priority' => 100],
```

Comment:
If yes, the examples of pre-routing middlewares in `features.md` should be updated as well.